client/me/help/main.jsx

```diff
@@ -325,15 +315,15 @@ function planHasOnboarding( { productSlug } ) {
 	return planHasFeature( productSlug, FEATURE_BUSINESS_ONBOARDING );
 }
 
-export const mapStateToProps = ( state, ownProps ) => {
+export const mapStateToProps = ( state ) => {
 	const isEmailVerified = isCurrentUserEmailVerified( state );
 	const userId = getCurrentUserId( state );
 	const purchases = getUserPurchases( state, userId );
 	const isLoading = isFetchingUserPurchases( state );
 	const isBusinessPlanUser = some( purchases, planHasOnboarding );
-	const showCoursesTeaser = ownProps.isCoursesEnabled && isBusinessPlanUser;
```

Comment:
I guess this allows us to remove the `help/courses` flag altogether, right? Not a blocker for this PR of course 👍

Reply:
I'd intended to roll that into this PR, but wanted to keep it to just the help files for now. I'm hoping to deploy this before the holidays so I'm being paranoid. 😅 I'll spin up a new PR to remove it from the config.